.github/actions/gradle-cache/action.yml

```diff
@@ -5,4 +5,5 @@ runs:
   steps:
     - uses: gradle/gradle-build-action@v2
       with:
-        cache-read-only: ${{ github.ref != 'refs/heads/dev' }}
+        cache-write-only: ${{ github.ref_name == 'dev' }}
```

Comment:
Good to know there is also a short ref name 😅

Reply:
There is a nice list of all available variables: [github-context](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) 🍻